From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Wed, 21 Jun 2023 16:35:33 -0300
Subject: Preserve debug info

Forwarded: not-needed
Last-Updated: 2023-06-20
---
 Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile b/Makefile
index 6a1b85c..473cbca 100644
--- a/Makefile
+++ b/Makefile
@@ -65,7 +65,6 @@ else
 endif
 
 ifndef GODEBUG
-	EXTRA_LDFLAGS += -s -w
 	DEBUG_GO_GCFLAGS :=
 	DEBUG_TAGS :=
 else
